docs/scos/dev/guidelines/performance-guidelines/spryker-elastic-computing/enable-queue-task-flow-in-new-relic.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Enable queue task flow in New Relic
+description: This guideline explains how to enable queue task flow in New Relic.
+last_updated: May 16, 2022
+template: concept-topic-template
+---
+
+By default, the `spryker/monitoring` module groups all New Relic transactions generated from a console command by the command’s name. For example, `ooms:check-timeout`.
+
+This strategy works in most cases, but you might want to override this behavior. For example, `queue:task:start {queue_name}`. With the default naming strategy, all the calls of the command with different queues are named `queue:task:start`. Transaction data is aggregated under the same name for all types of queue processors.
+
+In New Relic dashboard, this looks as follows:
+
+![New relic one]()
+
+> :info: Transaction data is harvested and can be queried by attribute with NRQL. === What are we trying to explain by this note?
```

Comment:
What are we trying to explain by this note?

Reply:
Although transactions are collected under the same name, the data of particular queue processing transaction is still accessible but through querying it with NRQL.  

=== What are we trying to explain by this note?
this needs to be removed.